### DISCUSSION TITLE
Add new type to validator schema.

### DISCUSSION THREAD

**Discussion by irwing-reza** [adonisjs/core#3963] [Help]
Hi,
Is there an example to add a new type to the validator schema ?

**Comment by RomainLanz** [comment]
Hey @irwing-reza! 👋🏻 

What do you have in mind when saying "new type"?

**Comment by irwing-reza** [comment]
So right now the schema has 
![image](https://user-images.githubusercontent.com/9361846/199805194-5bc66a9b-4eb8-418b-bafd-3db2b71f6792.png)

I want to have something similar to DateType but that returns a regular Date type instead of Luxon DateTime

I would like to be able to do something like

```
schema.regularDate.optional()
```
